### ISSUE TITLE
Moirai ↔ v18.4.0

### ISSUE THREAD

**Issue by fleskesvor** [itchio/itch-compatibility-watchlist#44]
:rotating_light: game [Moirai](https://hypernexus.itch.io/moirai) is broken for me.

:book: Here's the complete [debug log](https://gist.github.com/c2ba67b88e8f9ff7f5348228e05278b1).

:running: The shell script runMoirai sets the library path, works, and should be preferred by the app.

**Comment by fasterthanlime** [comment]
I agree! The app does prefer shell scripts.. but only those that end in `.sh`. That should be fixed :)
